project/AppDependencies.scala

```diff
@@ -5,20 +5,23 @@ import sbt._
 
 object AppDependencies {
 
+  private val bootstrapVer: String = "7.15.0"
+  private val mongoVer: String = "1.1.0"
+
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"         % "7.13.0",
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-work-item-repo-play-28" % "0.74.0",
-    "uk.gov.hmrc"             %% "agent-mtd-identifiers"             % "0.61.0-play-28",
-    "uk.gov.hmrc"             %% "cluster-work-throttling"           % "8.3.0-play-28",
-    "uk.gov.hmrc"             %% "agent-kenshoo-monitoring"          % "4.8.0-play-28",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"         % bootstrapVer,
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-work-item-repo-play-28" % mongoVer,
+    "uk.gov.hmrc"             %% "agent-mtd-identifiers"             % "1.2.0",
+    "uk.gov.hmrc"             %% "cluster-work-throttling"           % "8.5.0-play-28",
+    "uk.gov.hmrc"             %% "agent-kenshoo-monitoring"          % "5.4.0",
     "uk.gov.hmrc"             %% "crypto-json-play-28"               % "7.3.0",
     "com.github.blemale"      %% "scaffeine"                         % "4.0.1",
     "joda-time"               %  "joda-time"                         % "2.12.1",
     "net.codingwell"          %% "scala-guice"                       % "4.2.9"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"     % "7.13.0"        % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"     % bootstrapVer        % "test, it",
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"    % "0.74.0"        % "test, it",
```

Comment:
you've not used `mongoVer` for the test deps

Reply:
I reverted to a previous version for tests because of a nasty class with MockFactory that we can deal with later.